src/parser/paladin/retribution/modules/core/Judgment.js

```diff
@@ -17,38 +21,73 @@ class Judgment extends Analyzer {
   divineStormConsumptions = 0;
   justicarsVengeanceConsumptions = 0;
   judgmentsApplied = 0;
+  judgmentsOverwritten = 0;
+  judgmentTargets = [];
 
   on_byPlayer_applydebuff(event) {
     const spellId = event.ability.guid;
     if (spellId !== SPELLS.JUDGMENT_DEBUFF.id) {
       return;
     }
+    // The event doesn't specify instance on the first target of that type
+    let targetInstance = event.targetInstance;
+    if (targetInstance === undefined) {
+      targetInstance = 1;
+    }
     this.judgmentsApplied += 1;
+    const judgmentTarget = { targetID: event.targetID, targetInstance: targetInstance, timestamp: event.timestamp };
+    this.judgmentTargets.push(judgmentTarget);
```

Comment:
Why are you tracking this manually now? Is getBuff tracking bugged?

Reply:
getBuff was returning true for all enemies (or at least those with same targetID) when only one had the debuff